plaso/cli/psort_tool.py

```diff
@@ -531,6 +531,11 @@ def ProcessStorage(self):
       storage_writer = (
           storage_factory.StorageFactory.CreateStorageWriterForFile(
               session, self._storage_file_path))
+      if not storage_writer:
+        logger.error(
```

Comment:
Why not raise BadConfigOption instead?

Reply:
Because this isn't what's done for other similar cases, see line 515.

We can other instances of this to raise in a follow up PR if you'd prefer that.

Reply:
ack, this is fine for now